test/functional/terminal/tui_spec.lua

```diff
@@ -135,14 +135,26 @@ describe('tui', function()
     feed_data('\022\022') -- ctrl+v
     feed_data('\022\013') -- ctrl+m
     screen:expect([[
-    {9:^G^V^M}{1: }                                           |
+    {11:^G^V^M}{1: }                                           |
     {4:~                                                 }|
     {4:~                                                 }|
     {4:~                                                 }|
     {5:[No Name] [+]                                     }|
     {3:-- INSERT --}                                      |
     {3:-- TERMINAL --}                                    |
-    ]])
+    ]], {
+      [1] = {reverse = true},
+      [2] = {background = 11},
+      [3] = {bold = true},
+      [4] = {foreground = 12},
+      [5] = {bold = true, reverse = true},
+      [6] = {background = 11},
+      [7] = {foreground = 130},
+      [8] = {foreground = 15, background = 1},
+      [9] = {foreground = 4},
+      [10] = {foreground = 2},
+      [11] = {foreground = 81},
```

Comment:
Better to add this to `terminal/helpers.lua:55` because (1) less repetition (2) no subtle difference between the meaning of test in the same file (now `{11}` means different things in different tests, unexpected when 1-10 means the same)